pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -394,6 +398,10 @@ message CommandSend {
     optional int32 num_messages = 3 [default = 1];
     optional uint64 txnid_least_bits = 4 [default = 0];
     optional uint64 txnid_most_bits = 5 [default = 0];
+
+    /// Add lowest and highest sequence id to support external sequence id
+    optional uint64 lowest_sequence_id = 6 [default = 0];
+    optional uint64 highest_sequence_id = 7 [default = 0];
```

Comment:
Why do we need 2 new fields here? Don't we just need to a new `last_sequence_id`? isn't `lowest_sequence_id` the same as `sequence_id` ?

Reply:
Yes, we can only add a new last_sequence_id, will fix.